### PR TITLE
Update DG for logic component

### PR DESCRIPTION
**Pull request by kahsuann**
AddressBookParser -> EventToryParser
DeleteCommand -> DeleteEventCommand (for this example, delete e/1)

### CODE REVIEW

File: docs/DeveloperGuide.md
Comment (kevin-pek) [inline docs/DeveloperGuide.md:1]: The url for the link `Logic.java` on line 87 should be updated to link to our repo.

**Review by kevin-pek — DISMISSED** [review]
Changes look good to me aside from one thing that was missed out!

**Comment by Jasperabez** [comment]
@kahsuann Diagram looks ok too
DeleteSequenceDiagram

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -101,9 +101,9 @@ The sequence diagram below illustrates the interactions within the `Logic` compo
...
-1. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `DeleteCommand`) which is executed by the `LogicManager`.
-1. The command can communicate with the `Model` when it is executed (e.g. to delete a person).<br>
+1. When `Logic` is called upon to execute a command, it is passed to an `EventToryParser` object which in turn creates a parser that matches the command (e.g., `DeleteCommandParser`) and uses it to parse the command.
+1. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `DeleteEventCommand`) which is executed by the `LogicManager`.
+1. The command can communicate with the `Model` when it is executed (e.g. to delete an event).<br>
    Note that although this is shown as a single step in the diagram above (for simplicity), in the code it can take several interactions (between the command object and the `Model`) to achieve.
 1. The result of the command execution is encapsulated as a `CommandResult` object which is returned back from `Logic`.
```
Comment (Jasperabez) [inline docs/DeveloperGuide.md:109]: Can perhaps add in add note here talking on how deleting specific categories (event, vendor) e.g `DeleteEventCommand` is inheriting from `DeleteCommand`

**Review by Jasperabez — CHANGES_REQUESTED** [review]
LGTM but perhaps add in the inheritance part as a note. A

**Comment by Jasperabez** [comment]
would be better if the note was a separate section like the parser. thoughts?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -101,9 +101,9 @@ The sequence diagram below illustrates the interactions within the `Logic` compo
 
 How the `Logic` component works:
 
-1. When `Logic` is called upon to execute a command, it is passed to an `AddressBookParser` object which in turn creates a parser that matches the command (e.g., `DeleteCommandParser`) and uses it to parse the command.
-1. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `DeleteCommand`) which is executed by the `LogicManager`.
-1. The command can communicate with the `Model` when it is executed (e.g. to delete a person).<br>
+1. When `Logic` is called upon to execute a command, it is passed to an `EventToryParser` object which in turn creates a parser that matches the command (e.g., `DeleteCommandParser`) and uses it to parse the command.
+1. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `DeleteEventCommand` which inherits from `DeleteCommand`) which is executed by the `LogicManager`.
+1. The command can communicate with the `Model` when it is executed (e.g. to delete an event).<br>
```
Comment (C5hives) [inline docs/DeveloperGuide.md:106]: Numbering is broken :(

**Review by Jasperabez — APPROVED** [review]
Good work! LGTM

**Pull request merged**